### DISCUSSION TITLE
Alphanumeric Map Pins

### DISCUSSION THREAD

**Discussion by AmbushXXVI** [javalent/obsidian-leaflet#289] [Ideas]
When using this for gaming/dungeon maps, it would be helpful to have a somewhat generic pin (i.e. a bordered circle/square/default map pin) that contained a string. Wouldn't even need to be sequential (though that's a nice to have). I've attached an example below, which, technically I could do, but then I'd need a custom image and token for each number, and that seems painful. Unless I'm an idiot and you can already do this and I haven't figured it out yet. 

![image](https://user-images.githubusercontent.com/44707760/158467538-87294066-cba1-48bd-98cf-65285b731ab6.png)

**Comment by valentine195** [comment]
You cannot do this yet, good idea though, I’ll think about how to do it.

**Reply by AmbushXXVI** [reply]
Bummer it's not possible yet, but hopefully it's a thing that can happen. Thanks for the reply.

**Comment by oyvindsolstad** [comment]
Could you use these in any way? In Unicode all number from 1 up to 50 are a character. Example: ㉔  or ㊿

http://xahlee.info/comp/unicode_circled_numbers.html
